isushibsiteaccesssync.module

```diff
@@ -0,0 +1,218 @@
+<?php
+
+/**
+ * @file
+ * Syncs the access list with Active Directory group(s).
+ */
+
+
+/**
+ * Implements hook_menu().
+ */
+function isushibsiteaccesssync_menu() {
+  $items['admin/config/people/isushibsiteaccesssync'] = array(
+    'title' => 'Configure AD Sync',
+    'description' => 'Edit settings for Active Directory sync.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('isushibsiteaccesssync_settings'),
+    'access arguments' => array('administer shibboleth site access ad sync'),
+    'file' => 'isushibsiteaccesssync.admin.inc',
+  );
+
+  return $items;
+}
+
+
+/**
+ * Implements hook_cron().
+ */
+function isushibsiteaccesssync_cron() {
+  // Only run this once per day
+  if (variable_get('isushibsiteaccesssync_day_of_year', 0) == date('z') + 1) {
+    return;
+  }
+
```

Comment:
I would break the rest of this code into a separate function. As it is, when you call isushibsiteaccesssync_cron() from drush it will enforce the once a day requirement. I would assume if you are manually running it from drush you want it to go even if it already ran